src/bundle/Resources/public/js/scripts/admin.notifications.modal.js

```diff
@@ -245,15 +246,26 @@
     const attachActionsListeners = () => {
         const attachListener = (node, callback) => node.addEventListener('click', callback, false);
         const markAsButtons = doc.querySelectorAll('.ibexa-notifications-modal--mark-as');
-        const deleteButtons = doc.querySelectorAll('.ibexa-notifications-modal--delete');
+        const deleteButtons = doc.querySelectorAll('.ibexa-notifications-open-modal-button');
+        const confirmDeleteButton = doc.querySelector('.ibexa-notifications-modal--delete--confirm');
```

Comment:
nitpick - naming convention
```suggestion
        const deleteBtns = doc.querySelectorAll('.ibexa-notifications-open-modal-button');
        const confirmDeleteBtn = doc.querySelector('.ibexa-notifications-modal--delete--confirm');
```